### PR TITLE
Fix registering polyfill-php80 in the meta composer.json

### LINKED ISSUE #273: Installing symfony/polyfill as well as symfony/polyfill-php80 doesn't work
The functions never get defined. symfony/polyfill shouldn't have claimed to replace the polyfill if it didn't?

### PR DESCRIPTION
**Pull request by nicolas-grekas**
Fix #273

### CODE REVIEW

File: src/Php80/bootstrap.php [hunk 1]
```diff
@@ -20,23 +20,23 @@
 }
 
 if (!function_exists('fdiv')) {
-    function fdiv(float $dividend, float $divisor): float { return p\Php80::fdiv($dividend, $divisor); }
```
Comment (GrahamCampbell) [inline src/Php80/bootstrap.php:23]: Could we conditionally define these functions so that on PHP 7 they have the types?

Comment (nicolas-grekas) [reply src/Php80/bootstrap.php:23]: yes, updated.

**Comment by nicolas-grekas** [comment]
Wait, this is wrong. The php8 polyfill is not compatible with PHP5.

**Comment by stof** [comment]
Then we should not gave the `replace` rule either in the root composer.json, as it does not provide the feature.

**Comment by nicolas-grekas** [comment]
yes.it.does.

**Comment by stof** [comment]
@nicolas-grekas I don't understand your answer. Without this PR, installing `symfony/polyfill` does not provide the features as they are not autoloaded.

**Comment by nicolas-grekas** [comment]
The PHP 8 polyfill supports only PHP >= 7.

For this reason, on PHP 5, the root polyfill package does not define the functions of the php8 one.
On PHP 7, these functions are declared as expected.

See #234 for how this works.

**Comment by GrahamCampbell** [comment]
But there is still a bug. The polyfill package incorrectly claims to replace it, so if I'm on PHP 7 and the main polyfill package gets loaded and a library requires the php8 polyfill, then the library gets broken...

**Comment by GrahamCampbell** [comment]
Oh, wait. I see there's a dedicated bootstrap file for this case. Sorry for the confusion!
